apps/shorturl/kinds/shorturl.cue

```diff
@@ -0,0 +1,16 @@
+package kinds
+
+shorturl: {
+	kind:		"ShortURL"
+	pluralName:	"ShortURLs"
+	group:      "shorturl.grafana.app"
```

Comment:
`group` is now determined by the manifest's computed group from the `groupOverride` or the `appName`, so you no longer need to set this in the kind.